catroid/src/org/catrobat/catroid/physics/PhysicsLook.java

```diff
@@ -59,6 +63,7 @@ public void setLookData(LookData lookData) {
 		super.setLookData(lookData);
 		PhysicsWorld physicsWorld = ProjectManager.getInstance().getCurrentProject().getPhysicsWorld();
 		physicsWorld.changeLook(physicsObject, this);
+		updatePhysicsObjectState(true);
```

Comment:
Would you like me to remove the parameter from that method's implementation? I don't know what whoever implemented it at the time was thinking so I'm somewhat hesitant to do so.